README.md

```diff
@@ -10,10 +11,21 @@ To run the `dev` server:
 5. Go to the port specified in the output
 6. To stop the environment, run `deactivate`
 
+On Windows:
+### Development environment
+1. Navigate to the project directory in terminal
+2. Create a virtual environment using: `python3 -m venv env`
+3. Run `.\env\Scripts\activate`. This will start the Python virtual environment.
+4. Update pip in the virtual environment by running the following command: `python -m pip install --upgrade pip`
+5. Install Flask in the virtual environment by running the following command: `python -m pip install flask`
+6. In the environment, run `python3 main.py` to run the server in the `main.py` file.
+7. Go to the port specified in the output
+8. To stop the environment, Press CTRL+C to quit
```

Comment:
Minor: I'm not on windows, but ctrl+c is to stop the dev server, yes, but the virtual python environment is still running. To get your terminal back, I have to run `deactivate`. If this is also the case on Windows let's update this, if not never mind.

Reply:
You're right, fixed.